rct229/reports/utils.py

```diff
@@ -71,7 +74,7 @@ def _count_results(outcomes):
             elif type(result) is dict and any(
                 [
                     key.startswith("INVALID_") and key.endswith("_CONTEXT")
-                    for key in result.keys()
```

Comment:
Why do we need to update this line?

Reply:
This is because we don't need ```.keys()``` to iterate a dictionary. I believe we removed all the ```.keys()``` in the past, but for some reason this file still had ```.keys()```. So, I deleted it.